### PR TITLE
[New] Add dependent-dropdown@1.4.8 w/ git auto-update

### LINKED ISSUE #8226: [Request] Add dependent-dropdown
**Library name:** dependent-dropdown
**Git repository url:** https://github.com/kartik-v/dependent-dropdown

### PR DESCRIPTION
**Pull request by sashberd**
close #8226, cc @kartik-v

Pull request for issue: #8226
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kartik-v/dependent-dropdown
 * Official website (optional, not the repository): http://plugins.krajee.com/dependent-dropdown
 * NPM package url (optional): https://www.npmjs.com/package/dependent-dropdown
 * License and its reference:  BSD-3-Clause
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 12
   - Count of stars: 142
   - Count of forks: 50
 * NPM download stats (optional):
   - Downloads in the last day: 1
   - Downloads in the last week: 4
   - Downloads in the last month: 38

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@extend1994 @maruilian11 
Please help review this PR, Thank you.

**Comment by sashberd** [comment]
@PeterDaveHello You are right. The file was removed. FYR

File: ajax/libs/dependent-dropdown/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kartik-v/dependent-dropdown.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "+(css|js)/*dependent-dropdown*",
+          "js/locales/(!LANG)*",
```
Comment (sufuf3) [inline ajax/libs/dependent-dropdown/package.json]: @sashberd Would you try this package from npm? https://github.com/isaacs/node-glob#glob-primer (`npm i glob`)
This is a tool that you can know that how to get the files depend on the pattern.

When you work in terminal, you can enter into `node` environment after you `git clone dependent-dropdown` and change directory to `dependent-dropdown`.
Here are the demo:
```js
$ node
> glob = require('glob')
{ [Function: glob]
  sync: { [Function: globSync] GlobSync: [Function: GlobSync] },
  GlobSync: [Function: GlobSync],
  glob: [Circular],
  hasMagic: [Function],
  Glob:
   { [Function: Glob]
     super_:
      { [Function: EventEmitter]
        EventEmitter: [Circular],
        usingDomains: true,
        defaultMaxListeners: 10,
        init: [Function],
        listenerCount: [Function] } } }
> glob.sync('js/locales/(!LANG)*')
[]
> glob.sync('js/**/*')
[ 'js/dependent-dropdown.js',
  'js/dependent-dropdown.min.js',
  'js/locales',
  'js/locales/de.js',
  'js/locales/es.js',
  'js/locales/fr.js',
  'js/locales/it.js',
  'js/locales/LANG.js',
  'js/locales/pl.js',
  'js/locales/pt.js',
  'js/locales/ru.js' ]
```

Would you try to fix this?
I think we just need to exclude the `js/locales/LANG.js',` :)

Thanks

Comment (sashberd) [reply ajax/libs/dependent-dropdown/package.json]: @sufuf3 I used this one in order to check my patterns

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR
As I said in previous comment I have not option on Windows check patterns like you with glob, thus I  am using online tools.

**Comment by PeterDaveHello** [comment]
@sashberd to have the more efficient review process, please let some other else to request the review from me, you'll need the others help you review PRs first, thanks.

**Comment by sufuf3** [comment]
Hi @sashberd Would you like to add CDNJS badge on this library?
Like https://github.com/jaredreich/pell/pull/52
We use https://shields.io/ to generate the badge.
The benefits of adding badge is help developers know the lib which can be used via CDN.

This will be a great work.
Would you try?

PS. Also, you can sign in https://24pullrequests.com/, this is a cool event. Just share with you.

Thank you for being active-contributor.

**Pull request merged**